README.md

```diff
@@ -1,6 +1,6 @@
 # AWS JWT Verify
 
-**Node.js** library for **verifying** JWTs signed by **Amazon Cognito**, and any **OIDC-compatible IDP** that signs JWTs with **RS256**.
+**Node.js** library for **verifying** JWTs signed by **Amazon Cognito**, and any **OIDC-compatible IDP** that signs JWTs with **RS256** / **RS512**.
```

Comment:
Should we support RS384? Because it is now so easy to do so, just another entry in the enum

Reply:
Agreed, pushed a new commit that added RS384 support as well